test/http-context.test.js

```diff
@@ -74,6 +75,24 @@ describe('HttpContext', function() {
         expectedValue: '000123'
       }));
     });
+
+    describe('arguments with custom type', function() {
+      Dynamic.define('CustomType', function(val) {
+        return JSON.parse(val);
+      });
+
+      it('should coerce dynamic type with string prop into object', givenMethodExpectArg({
+        type: 'CustomType',
+        input: JSON.stringify({ stringProp: 'string' }),
+        expectedValue: { stringProp: 'string' }
+      }));
+
+      it('should coerce dynamic type with int prop into object', givenMethodExpectArg({
+        type: 'CustomType',
```

Comment:
I believe this test will break before the patch. There is no functional difference between `CustomType` and `customType` in this regard, both will fail. Only `customtype` will not fail at the moment.